.github/workflows/bumpVersions.yml

```diff
@@ -33,18 +39,29 @@ jobs:
       - name: Install updatecli
         uses: updatecli/updatecli-action@v2
         with:
-          version: v0.54.0
+          version: v0.66.0
+
+      - name: Set up Homebrew
+        id: set-up-homebrew
+        uses: Homebrew/actions/setup-homebrew@abcbd5d83bbe1a69c8da087467fcd132ca0c7a27
+
+      - name: Install htmlq
+        run: |
+          brew install htmlq
 
       - name: Set required environment variables
         run: |
           echo "TOMCAT_MAJOR=${{ matrix.tomcat_major }}" >> $GITHUB_ENV
-          echo "TOMCAT_VERSION=$(jq -r '.tomcat_version' tomcat${{ matrix.tomcat_major }}.json)" >> $GITHUB_ENV
-          echo "TCNATIVE_VERSION=$(jq -r '.tcnative_version' tomcat${{ matrix.tomcat_major }}.json)" >> $GITHUB_ENV
```

Comment:
this was like a loophole because updatecli was feed from the same files it was going to update later (and was also causing the pr title to contain the PREVIOUS version)